### PR TITLE
chore: rename `explorer` settings to `arkscan`

### PR DESCRIPTION
**Pull request by alfonsobries**
## Summary

explorer is no longer valid and may be confusing for someone that doesnt know that the site used to be named explorer, personally sometimes I need a few seconds to remember the name of the config file when I need to update something

image

## Checklist

-   [ ] I checked my UI changes against the design and there are no notable differences
-   [ ] I checked my UI changes for any responsiveness issues
-   [ ] I checked my UI changes in light AND dark mode
-   [ ] I checked my (code) changes for obvious issues, debug statements and commented code
-   [ ] I opened a corresponding card on Clickup for any remaining TODOs in my code
-   [ ] I added a short description on how to test this PR _(if necessary)_
-   [ ] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [ ] Ready to be merged

### CODE REVIEW

**Comment by alexbarnsley** [comment]
Mici can you review this when you're back please :oleglala:

**Comment by alexbarnsley** [comment]
@ItsANameToo

**Comment by ItsANameToo** [comment]
this should be alright, matches the `.env` variables as well with this change as those are all `ARKSCAN_`

File: tests/Feature/Console/KernelTest.php [hunk 1]
```diff
@@ -9,7 +9,7 @@
 const EVERY_MINUTE = '* * * * *';
 
 it('schedules the `scout:index-models` command correctly if enabled', function () {
-    Config::set('explorer.scout.run_jobs', true);
+    Config::set('arkscan.scout.run_jobs', true);
```
Comment (ItsANameToo) [inline tests/Feature/Console/KernelTest.php:12]: interesting that this test worked while the config changed 🤔

Comment (ItsANameToo) [reply tests/Feature/Console/KernelTest.php:12]: apparently the config check in `Kernel.php` wasn't updated so it referenced an old one with a default value

**Review by ItsANameToo — APPROVED** [review]

**Pull request merged**